src/api/app/controllers/webui/obs_factory/staging_projects_controller.rb

```diff
@@ -38,6 +38,8 @@ def show
           @staging_project = ::ObsFactory::StagingProjectPresenter.new(@staging_project)
           # For the breadcrumbs
           @project = @distribution.project
+          build_id = Backend::Api::Published.build_id(@staging_project.name, 'images')
+          @checks = Status::Check.includes(:repository_publish).where(status_repository_publishes: { build_id: build_id })
```

Comment:
:thought_balloon: Why not going from `Repository`. You know it...

Reply:
Done